CHANGELOG.md

```diff
@@ -19,6 +59,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Send event id to group building block if event is published or group id has been updated. [#935](https://github.com/rokwire/events-manager/issues/935)
 - Update the style of create event button. [#967](https://github.com/rokwire/events-manager/issues/967)
 - Update the style of the footer. [#968](https://github.com/rokwire/events-manager/issues/968)
+- Update the style of buttons on the event detail page. [#984](https://github.com/rokwire/events-manager/issues/984)
+- Add elements to the footer. [#969](https://github.com/rokwire/events-manager/issues/969)
```

Comment:
Confused on why theres some changes here in the 3.4.0 release. Saw an issue on this https://github.com/rokwire/events-manager/issues/1033. 
Maybe will be fixed later? But it would be nice to fix before a release..

Reply:
These issues should have been part of the Unreleased section earlier. So, yes, these need to be moved to appropriate sections under the 3.5.0 release.

Reply:
fixed

Reply:
Thanks.